scripts/build-products.js

```diff
@@ -36,10 +36,26 @@ const fetchProductTypes = async cover => {
   return productTypes;
 };
 
+const createLogoDict = async logosDir => {
+  const dirents = await readdir(logosDir, { withFileTypes: true });
+  const filenames = dirents.map(dirent => dirent.name);
+
+  const map = filenames.reduce((acc, filename) => {
+    const name = filename.substring(filename.indexOf('-') + 1);
```

Comment:
🤔 Shouldn't the logo name be the full filename?

Reply:
But the svg files' names are still containing the `00x` prefix in the repo. We need the full name of the logo file for each product, which is now changed with the prefix.

Reply:
What I'm trying to say is that the map should be `productId` => `logo-filename`.

Reply:
It is only the source files that have the `00x` prefix, after reading them they are never used anymore. This means that the `001-oneinch.svg` file will be exported as `oneinch.svg`. So we also need to remove the `00x` prefix [here](https://github.com/NexusMutual/sdk/pull/61/files#diff-d3a595c6259d34e084822631224dc0c2fcb4ab7ccc7c694339ce435121114bc5R48) and [here](https://github.com/NexusMutual/sdk/pull/61/files#diff-d3a595c6259d34e084822631224dc0c2fcb4ab7ccc7c694339ce435121114bc5R30).

Reply:
after our discussion, I understand now that they are in fact inline with the names that are exported so it should be fine.